proto/tabletmanagerservice.proto

```diff
@@ -153,7 +159,10 @@ service TabletManager {
   rpc ReplicaWasPromoted(tabletmanagerdata.ReplicaWasPromotedRequest) returns (tabletmanagerdata.ReplicaWasPromotedResponse) {};
 
   // SetMaster tells the replica to reparent
-  rpc SetMaster(tabletmanagerdata.SetMasterRequest) returns (tabletmanagerdata.SetMasterResponse) {};
+  rpc SetMaster(tabletmanagerdata.SetPrimaryRequest) returns (tabletmanagerdata.SetPrimaryResponse) {};
+
+  // SetMaster tells the replica to reparent
+  rpc SetPrimary(tabletmanagerdata.SetPrimaryRequest) returns (tabletmanagerdata.SetPrimaryResponse) {};
```

Comment:
Over on the tabletmanager side I chose to call this `SetReplicationSource` instead of `SetPrimary`. That seemed more apt in the context of replication.
So.. cluster management level master -> primary
MySQL replication level master -> replication source.
Can we change this to be consistent?

Reply:
done